Source/Flow/Private/AddOns/FlowNodeAddOn_PredicateAND.cpp

```diff
@@ -0,0 +1,53 @@
+// Copyright https://github.com/MothCocoon/FlowGraph/graphs/contributors
+
+#include "AddOns/FlowNodeAddOn_PredicateAND.h"
+
+#include UE_INLINE_GENERATED_CPP_BY_NAME(FlowNodeAddOn_PredicateAND)
+
+UFlowNodeAddOn_PredicateAND::UFlowNodeAddOn_PredicateAND()
+	: Super()
+{
+#if WITH_EDITOR
+	NodeStyle = EFlowNodeStyle::Logic;
+	Category = TEXT("Composite");
+#endif
+}
+
+EFlowAddOnAcceptResult UFlowNodeAddOn_PredicateAND::AcceptFlowNodeAddOnChild_Implementation(const UFlowNodeAddOn* AddOnTemplate) const
+{
+	if (IFlowPredicateInterface::ImplementsInterfaceSafe(AddOnTemplate))
```

Comment:
Here we see the AddOn deciding eligibility for its potential child AddOns, which is the opposite direction from the Parent-side version mentioned in a previous comment above.

In this case, the AddOn or Node parent knows what child AddOns it supports and filters them out with this function.